bps/clustering/DRP-recalibrated.yaml

```diff
@@ -0,0 +1,58 @@
+# This is a prescription for quantum clustering with BPS, suitable for any
+# concrete pipeline based on the "DRP-recalibrated" pipeline.  It also includes
+# clustering for analysis_drp plots.
+#
+# Use it by adding:
+#
+#   includeConfigs:
+#    - ${DRP_PIPE_DIR}/bps/clustering/DRP-recalibrated.yaml
+#
+# (with no outer indentation) to your BPS config file.
+
+clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.dimension_clustering
+cluster:
```

Comment:
General comment: starting here, it seems like the rest of this file is using a tab indent of 4 spaces, whereas I think this needs to be 2 spaces (as with the other YAMLs in this PR).

Reply:
The yaml linter configuration just requires that we be consistent within a single file. Maybe we need to be a bit more draconian.